### PR TITLE
coderbyte first_revese solution added

### PR DESCRIPTION
**Pull request by tbhaxor**
Coderbyte problem solution added
**Problem :** first reverse

@AdiChat 
@iattempt 
@arnavb 

ps : I will commit all the changes to this branch

### CODE REVIEW

**Comment by tbhaxor** [comment]
Also i am not understanding this new style of test
@iattempt will u please help me in passing this test by changing whatever u want in my PR :confused:

**Comment by InfiniteCoder** [comment]
@tbhaxor You can ignore the "travis CI build failed" error for now. It actually compiles all the programs, and checks if there are any compilation errors and warnings. As currently, many previous programs have issues, it gives errors. You can ignore it for now, or if you wish, you can help to fix the issues, by having a look at the [logs here](https://travis-ci.org/OpenGenus/cosmos) and fixing the programs which have issues.

File: code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+string reverseIt(char* str)
+{
+	string st(str);
+	reverse(st.begin(), st.end());
+	return st;
```
Comment (arnavb) [inline code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp]: You could also do (Without having to call the reverse function and including algorithm):

```C++
return string{st.rbegin(), st.rend()};
```

File: code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+int main(int argc, char* argv[])
+{
+	cout<<"Reversing "<<argc - 1<<" strings"<<endl;
+	for (int i = 1; i < argc; i++)
```
Comment (arnavb) [inline code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp]: Use preincrement, just cause.

File: code/online_challenges/src/coderbyte/first_reverse/readme.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+# First Reverse Program
```
Comment (arnavb) [inline code/online_challenges/src/coderbyte/first_reverse/readme.md]: I think the README can be a bit more detailed, and the word readme should be capitalized completely.

File: code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp (hunk 1 above)
Comment (tbhaxor) [reply code/online_challenges/src/coderbyte/first_reverse/firstReverse.cpp]: but i used a standard cpp function @arnavb

**Pull request closed without merging**